projects/batfish/src/org/batfish/grammar/cisco/Cisco_aaa.g4

```diff
@@ -421,7 +421,7 @@ aaa_authorization_commands
    COMMANDS level = DEC?
```

Comment:
Where are the unit tests? I don't fully understand the impact of this change. Also, please make it clear what flavor of Cisco your example lines are coming from.